modules/wolfgame.py

```diff
@@ -5166,6 +5055,7 @@ def transition_night(cli):
         else:
             pm(cli, v_ghost, "You are a \u0002vengeful ghost\u0002.")
         pm(cli, v_ghost, who.capitalize() + ": " + ", ".join(pl))
+        debuglog("GHOST: {0} (Targets: {1}) Players: {2}".format(v_ghost, who.capitalize(), " ".join(pl)))
```

Comment:
1. Why is "targets" capitalized?
2. Not sure how useful listing the players is. I guess it can be useful but I don't know. Anyway, if they're listed, I think it should say `- players:` and the actual players should be comma-separated.